libraries/ts-command-line/src/test/__snapshots__/CommandLineParameter.test.ts.snap

```diff
@@ -550,55 +548,54 @@ exports[`CommandLineParameter prints the action help 1`] = `
                           [--string-with-default TEXT]
                           [--string-with-undocumented-synonym TEXT]
                           [-l LIST_ITEM]
-                          
 
 a longer description
 
-Optional arguments:
-  -h, --help            Show this help message and exit.
+optional arguments:
```

Comment:
This inconsistent formatting looks bad; if the base is changing should we change to match?

Reply:
I don't think so. Beginning sentences with a lower case letter looks objectively worse.  I checked a few other popular CLI tools (`tsc`, `eslint`, `PowerShell`, `gcc`) and their `--help` always uses proper English capitalization. Doing so also encourages grammatically complete sentences, which may inspire better documentation. Notably `python --help` does show unprofessional lowercase capitalization, however Python's own `pip --help` uses caps like all the other examples.

I believe this behavior is customizable by implementing our own `argparse` help formatter.

Reply:
The API allows us to suppress the default `--help` parameter and define it ourselves.

I created an issue https://github.com/nodeca/argparse/issues/175 requesting a more general solution, however the NPM package hasn't been updated in 3 years, so we'll see if they reply. 😄